src/Workspaces/Core/Portable/Remote/RemoteArguments.cs

```diff
@@ -37,35 +37,47 @@ public FindReferencesSearchOptions Rehydrate()
         }
     }
 
+    /// <summary>
+    /// Note: for purposes of Equality/Hashing, all that we use is the underlying SymbolKey.  That's because nearly all
+    /// IDE features only care if they're looking at the same symbol, they don't care if the symbol came from a
```

Comment:
"nearly all" -- can we just make this not implement IEquatable and the features have to be explicit about the equality they need?

Reply:
yes.  that seems very reasonable.

Reply:
done.